tgg/seqr_scripts/seqr_sample_qc.py

```diff
@@ -313,23 +324,24 @@ def run_hail_sample_qc(mt: hl.MatrixTable, data_type: str) -> hl.MatrixTable:
 def main(args):
 
     hl.init(log="/seqr_sample_qc.log")
-
+    hl._set_flags(no_whole_stage_codegen="1")
```

Comment:
might need a comment stating what version of hail currently needs this flag

Reply:
Good point! thoughts on?
```suggestion
    hl._set_flags(no_whole_stage_codegen="1") #Flag needed for hail 0.93, may be able to remove in future release.
```

Reply:
```suggestion
    hl._set_flags(no_whole_stage_codegen="1") #Flag needed for hail 0.2.93, may be able to remove in future release.
```

Yeah, sounds good - just missing the 2.